models/payment_db.go

```diff
@@ -19,6 +19,7 @@ type PaymentResourceDataDB struct {
 	CompletedAt             time.Time      `bson:"completed_at,omitempty"`
 	CreatedAt               time.Time      `bson:"created_at,omitempty"`
 	CreatedBy               CreatedByDB    `bson:"created_by"`
+	IpAddress               string         `bson:"ip_address"`
```

Comment:
Variable should be named `IPAddress`

https://github.com/golang/go/wiki/CodeReviewComments#initialisms